paderbox/transform/module_fbank.py

```diff
@@ -20,31 +20,44 @@ def __init__(
             self,
             sample_rate: int,
             fft_length: int,
-            n_mels: Optional[int] = 40,
+            n_mels: int,
             fmin: Optional[int] = 50,
             fmax: Optional[int] = None,
             log: bool = True,
             eps: float = 1e-18,
+            *,
+            warping_fn=None,
+            independent_axis=0,
     ):
         """
         Transforms linear spectrogram to (log) mel spectrogram.
 
         Args:
+            n_mels: number of filters to be applied
             sample_rate: sample rate of audio signal
             fft_length: fft_length used in stft
-            n_mels: number of filters to be applied
             fmin: lowest frequency (onset of first filter)
             fmax: highest frequency (offset of last filter)
             log: apply log to mel spectrogram
+            eps:
+            warping_fn: function to (randomly) remap fbank center frequencies
```

Comment:
Maybe you could also add a sentence about `independent_axis`here, e.g., something similar to the description of `size` in `get_fbanks`.